frontend/src/container/Header.jsx

```diff
@@ -33,12 +34,31 @@ import { Link } from 'react-router-dom'
 import {
   PROFILE,
   ComposedIcon,
-  CUSTOM_EVENT
+  CUSTOM_EVENT,
+  TLM_CORE_EVENT_TYPE as TLM_CET,
+  TLM_ENTITY_TYPE as TLM_ET,
+  TracimComponent
 } from 'tracim_frontend_lib'
 
 const qs = require('query-string')
 
-class Header extends React.Component {
+export class Header extends React.Component {
+  constructor (props) {
+    super(props)
+
+    props.registerLiveMessageHandlerList([
+      { entityType: TLM_ET.USER, coreEntityType: TLM_CET.MODIFIED, handler: this.handleUserModified }
+    ])
+  }
+
+  handleUserModified = data => {
+    const { props } = this
+    if (props.user.userId !== data.user.user_id) return
+
+    if (props.user.lang !== data.user.lang) props.dispatch(setUserLang(data.user.lang))
+    else props.dispatch(updateUser(data.user))
```

Comment:
with the PR https://github.com/tracim/tracim/pull/3194, this should be handled by the new component ReduxTlmDispatcher

You will have to update this PR once https://github.com/tracim/tracim/pull/3194 is merged

Reply:
update: the pr is merged